localstack/dev/run/__main__.py

```diff
@@ -281,6 +285,25 @@ def run(
         configurators.append(DependencyMountConfigurator(host_paths=host_paths))
     if develop:
         configurators.append(ContainerConfigurators.develop)
+    if dev_extensions:
+        if pro:
+            try:
+                from localstack_ext.extensions.bootstrap import (
+                    run_on_configure_host_hook,
+                    run_on_configure_localstack_container_hook,
+                )
+
+                # collect dev extensions
+                run_on_configure_host_hook()
+
+                # create stub container with configuration to apply
+                c = Container(container_config=container_config)
```

Comment:
I`m not pleased about wrapping the configuration in an empty `Container` but it re-uses the code which is likely more important.

Reply:
no i think we could refactor the code to use take the config instead of the container :+1: but i'll leave it up to you if you want to do it or not :-)